src/App.jsx

```diff
@@ -13,34 +13,83 @@
   'Jam',
   'Garlic',
 ];
+const SORT_FIELD_ALPHABETICCALY = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPreparedGoods(goods, sortField, isReversed) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  preparedGoods.sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_FIELD_ALPHABETICCALY:
```

Comment:
The typo in the constant name `SORT_FIELD_ALPHABETICCALY` is repeated here. Ensure consistency by using the corrected name `SORT_FIELD_ALPHABETICALLY`.